docs/release-notes.rst

```diff
@@ -3,6 +3,23 @@
 Release Notes
 #############
 
+v0.2.4
+******
+
+Date released: 03-08-2020
+
+Non-breaking changes and improvements:
+
+- Improve EOF Exceptions (`#2103 <https://github.com/vyperlang/vyper/pull/2115>`_)
```

Comment:
```suggestion
- Improve EOF Exceptions (`#2115 <https://github.com/vyperlang/vyper/pull/2115>`_)
```

The numbers shown in the link text don't line up with the numbers in the actual links.